internal/http/server.go

```diff
@@ -47,6 +47,12 @@ const (
 	// DefaultShutdownTimeout - default shutdown timeout to gracefully shutdown server.
 	DefaultShutdownTimeout = 5 * time.Second
 
+	// DefaultIdleTimeout for idle inactive connections
+	DefaultIdleTimeout = 30 * time.Second
+
+	// DefaultReadHeaderTimeout for very slow inactive connections
+	DefaultReadHeaderTimeout = 30 * time.Second
```

Comment:
Even 10 seconds is a long time. If the client making the request cannot serve a header within 10s or we can't process it in that time for CPU starvation rejecting it is fine IMO. I am however fine starting conservative with 30s.

Reply:
Agreed, but I just didn't want an aggressive timeout.